src/site/_data/tagsData.json

```diff
@@ -1,253 +1,82 @@
-  "layout": {
-    "title": "Layout"
-  },
-  "lighthouse": {
-    "title": "Lighthouse",
-    "description": "Our latest news, updates, and stories about Lighthouse."
-  },
-  "media": {
-    "title": "Media"
-  },
-  "media-queries": {
-    "title": "Media Queries"
-  },
-  "memory": {
-    "title": "Memory",
-    "description": "Memory management on the web."
-  },
-  "metrics": {
-    "title": "Metrics"
-  },
-  "mini-apps": {
-    "title": "Mini apps",
-    "description": "Apps built with Web technologies but not running in the browser."
-  },
-  "mobile": {
-    "title": "Mobile"
-  },
-  "modules": {
-    "title": "Modules"
-  },
-  "monetization": {
-    "title": "Monetization"
-  },
-  "network": {
-    "title": "Network"
-  },
-  "node": {
-    "title": "Node"
-  },
-  "notifications": {
-    "title": "Notifications"
-  },
-  "offline": {
-    "title": "Offline"
-  },
-  "origin-trials": {
-    "title": "Origin Trials"
-  },
-  "payments": {
-    "title": "Payments"
-  },
-  "performance": {
-    "title": "Performance"
-  },
-  "permissions": {
-    "title": "Permissions"
-  },
-  "progressive-web-apps": {
-    "title": "Progressive Web Apps",
-    "description": "Our latest news, updates, and stories about Progressive Web Apps."
-  },
-  "privacy": {
-    "title": "Privacy"
-  },
-  "puppeteer": {
-    "title": "Puppeteer",
-    "description": "Our latest news, updates, and stories about Puppeteer."
-  },
-  "rendering": {
-    "title": "Rendering"
-  },
+  "3d": {},
+  "accessibility": {},
+  "amp": {},
+  "analytics": {},
+  "animations": {},
+  "audio": {},
+  "augmented-reality": {},
+  "canvas": {},
+  "capabilities": {},
+  "case-study": {},
+  "cast": {},
+  "chrome-ux-report": {},
+  "chrome-dev-summit": {},
+  "content-security-policy": {},
+  "cookies": {},
+  "cors": {},
+  "css": {},
+  "devices": {},
+  "devtools": {},
+  "dom": {},
+  "encryption": {},
+  "engineering-blog": {},
+  "feature-policy": {},
+  "file-system": {},
+  "fonts": {},
+  "forms": {},
+  "games": {},
+  "graphics": {},
+  "houdini": {},
+  "html": {},
+  "identity": {},
+  "images": {},
+  "install": {},
+  "intl": {},
+  "iot": {},
+  "javascript": {},
+  "layout": {},
+  "lighthouse": {},
+  "media": {},
+  "media-queries": {},
+  "memory": {},
+  "metrics": {},
+  "mini-apps": {},
+  "mobile": {},
+  "modules": {},
+  "monetization": {},
+  "network": {},
+  "node": {},
+  "notifications": {},
+  "offline": {},
+  "origin-trials": {},
+  "payments": {},
+  "performance": {},
+  "permissions": {},
+  "progressive-web-apps": {},
+  "privacy": {},
+  "puppeteer": {},
+  "rendering": {},
   "scale-on-web": {
-    "title": "Scale on web",
-    "description": "A collection of short case studies from India and Southeast Asia celebrating the success stories of Google partners across multiple aspects of modern web development. Learn how these partners achieved scale while building amazing web experiences and delighting their customers.",
     "hero": {
```

Comment:
We discussed making tags.yml the source of truth. We'll delete everything in this file, except for this one quirky thing. And if folks want to give tag pages heros, then we support that by unioning this object with tags.yml